css/style.css

```diff
@@ -0,0 +1,110 @@
+@import url("https://fonts.googleapis.com/css2?family=Manrope&display=swap");
+@import url(../css/variables.css);
+
+* {
+  font-family: "Manrope", sans-serif;
+}
+
+/*header  */
+#header {
+  font-size: 3rem;
+  max-height: 10rem;
+  box-shadow: 0 5px 15px rgba(0, 0, 0, 0.6);
+  color: var(--txt-color-dark);
+  background: var(--bg-primary);
+  /* width: 100%; */
+  position: sticky;
+  top: 0;
+  left: 0;
+  z-index: 10;
+}
+#navbar ul li {
+  list-style-type: none;
+  padding: 0 2rem;
+  font-size: 2rem;
+  position: relative;
+}
+
+#navbar ul li a:hover,
+#navbar ul li a.active {
+  color: var(--hover-color);
+}
+#navbar ul li a.active::after {
+  content: "";
+  position: absolute;
+  width: 5rem;
+  height: 4px;
+  background-color: var(--hover-color);
+  bottom: -6px;
+  right: 23px;
+}
+.menu {
+  position: relative;
+}
+
+#menu-list {
+  position: absolute;
+  top: 45px;
+  background: #e9e9e9;
```

Comment:
as you are using variables for colors here also for `background` you could use variable